script.js

```diff
@@ -231,3 +231,30 @@ formCheck.onsubmit = (e) => {
   }
   return true;
 };
+
+// preserve data in the browser
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const commentInput = document.getElementById('comment');
+
+document.getElementById('formSubmit').addEventListener('submit', () => {
+  const datas = {
+    name: nameInput.value.trim(),
+    email: emailInput.value.trim(),
+    comment: commentInput.value.trim(),
+  };
+  if (datas.name && datas.email && datas.comment) {
+    localStorage.setItem('storedData', JSON.stringify(datas));
+  }
+});
```

Comment:
- [ ] The user inputs are not supposed to be saved in the local storage only after the user has submitted the form. The save must be async. meaning as the user is filling(typing) at the same time the inputs are being saved in the local storage.
Hint: `You may consider removing this block of code in the submit event` :smile: 

![Screenshot from 2022-10-22 13-12-15](https://user-images.githubusercontent.com/97100091/197336273-72b5b0de-fb6d-4d92-90e5-fed3ee384c35.png)